src/Console/LintCommand.php

```diff
@@ -29,6 +31,7 @@ public function configure()
             ->addOption('severity', 's', InputOption::VALUE_REQUIRED, 'The maximum allowed error level.', 'warning')
             ->addOption('reporter', 'r', InputOption::VALUE_REQUIRED, 'The reporter to use.', 'console')
             ->addOption('display', 'd', InputOption::VALUE_REQUIRED, 'The violations to display, "' . self::DISPLAY_ALL . '" or "' . self::DISPLAY_BLOCKING . '".', self::DISPLAY_ALL)
+            ->addOption('error', 'e',  InputOption::VALUE_OPTIONAL, 'Throw syntax error when a template contains an invalid token.', false)
```

Comment:
I think that syntax-error is more explicit

```suggestion
            ->addOption('syntax-error', 'e',  InputOption::VALUE_OPTIONAL, 'Throw syntax error when a template contains an invalid token.', false)
```

Reply:
May I suggest `throw-syntax-errors`? Looks to me that we're ready to merge after that!